src/routes.js

```diff
@@ -35,24 +36,27 @@ export default (store) => {
    * Please keep routes in alphabetical order
    */
   return (
-    <Route path="/" component={App}>
-      { /* Home (main) route */ }
-      <IndexRoute component={Home}/>
+    <Router>
+      <Route path="/" component={RedirectToDefaultLocale} />
```

Comment:
I think `onEnter` is before `component`. And give you the option to redirect before enter on a component. https://github.com/rackt/react-router/blob/f5aa145d465d3d85eb63f80f09029c59b6db1b4a/docs/API.md#onenternextstate-replacestate